native/cocos/scene/Camera.cpp

```diff
@@ -160,7 +160,10 @@ void Camera::update(bool forceUpdate /*false*/) {
     if (_node->getChangedFlags() || forceUpdate) {
         _matView = _node->getWorldMatrix().getInversed();
         _forward.set(-_matView.m[2], -_matView.m[6], -_matView.m[10]);
-
+        Mat4 scaleMat{};
+        scaleMat.scale(_node->getWorldScale());
+        // remove scale
+        Mat4::multiply(scaleMat, _matView, &_matView);
```

Comment:
Why native implementation is different from ts version?

Reply:
native's dst is on the right, typescript is on the left. so it is consistent.

static void cc::Mat4::multiply(const cc::Mat4 &m1, const cc::Mat4 &m2, cc::Mat4 *dst)